xray/src/bin/merge_xray_quadtrees.rs

```diff
@@ -20,8 +24,171 @@ struct CommandlineArguments {
     input_directories: Vec<PathBuf>,
 }
 
+fn copy_images(input_directory: &Path, output_directory: &Path) {
+    if input_directory == output_directory {
```

Comment:
I think this equality only makes sense after `canonicalize()`ing the paths. Otherwise, relative and absolute paths of the same dir seem to be different.

Reply:
:+1: fixed